hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/HandleWorkflow.java

```diff
@@ -339,7 +339,11 @@ private void handleUserTransaction(
                 recordBuilder.status(validationResult.responseCodeEnum());
 
             } else {
-                feeAccumulator.charge(payer, fees);
+                if (authorizer.hasPrivilegedAuthorization(payer, transactionInfo.functionality(), txBody)
```

Comment:
@natanasow I think this will fix your issue. (https://github.com/hashgraph/hedera-services/pull/8555/files#r1327224884)